README.md

```diff
@@ -488,7 +488,10 @@ CameraRoll.iosGetImageDataById(internalID, true);
 | Name         | Type                    | Required   | Description                                          |
 | ------------ | ----------------------- | ---------- | ---------------------------------------------------- |
 | internalID   | string                  | Yes        | Ios internal ID 'PH://xxxx'.                         |
-| convertHeic  | boolean                 | False      | Whether to convert or not to JPEG image.             |
+| options      | PhotoConvertionOptions  | False      | Expects an options with the shape described below.   |
```

Comment:
I guess you mean 'options object" ? 🙂

Reply:
Yes, so I copied it from another place in the readme without reading it well :) Updated!